server/base/src/main/java/org/apache/accumulo/server/init/Initialize.java

```diff
@@ -748,24 +733,27 @@ private String getRootUserName(SiteConfiguration siteConfig, Opts opts) throws I
     if (opts.cliPassword != null) {
       return opts.cliPassword.getBytes(UTF_8);
     }
-    String rootpass;
-    String confirmpass;
+    char[] rootpass;
+    char[] confirmpass;
+    String strrootpass;
+    String strconfirmpass;
     do {
-      rootpass = getLineReader().readLine(
-          "Enter initial password for " + rootUser + getInitialPasswordWarning(siteConfig), '*');
+      rootpass = System.console().readPassword(
+          "Enter initial password for " + rootUser + getInitialPasswordWarning(siteConfig));
```

Comment:
```suggestion
     var rootpass = System.console().readPassword(
          "Enter initial password for " + rootUser + getInitialPasswordWarning(siteConfig));
```

Could do something like this for both passwords and drop the char[] objects above.

Reply:
Done.